src/runner.ts

```diff
@@ -156,8 +156,8 @@ async function runWorker(options: Options, logger: Logger): Promise<Status> {
     return options.force || errorCount === 0 ? Status.Ok : Status.LintError;
 }
 
-// tslint:disable-next-line:no-async-without-await
-async function runLinter(options: Options, logger: Logger): Promise<LintResult> {
+// tslint:disable-next-line:promise-function-async
```

Comment:
ok what... swapping one disabled rule for another is a red flag. seems this new rule contradicts an existing one. maybe this while PR should just be an option on `promise-function-async`?

@JKillian @suchanlee thoughts?

Reply:
I conclude that the `no-async-without-await` rule should allow `return`, because an async `return` returns a promise which is desirable. When you have an async function that `await`s nothing and `return`s nothing - that's a sign of a problem the rule should help you with. What do you think?